htp/htp_util.c

```diff
@@ -2536,3 +2536,54 @@ htp_uri_t *htp_uri_alloc(void) {
 char *htp_get_version(void) {
     return HTP_VERSION_STRING_FULL;
 }
+
+htp_status_t htp_header_has_token(const unsigned char *hvp, size_t hvlen, const unsigned char *value) {
```

Comment:
please provide documentation of the function

Reply:
ok